### ISSUE TITLE
Everything freeze

### ISSUE THREAD

**Issue by BI4LY** [Courseplay/courseplay#3066]
I have problem, when I start CP for truck to go unload combine, when he reach final destination where he has to wait for filling level game freeze, I can switch to other machine but only once, I can not get out of machine because game will freeze completly. Only when I click "stop driver" everything back to normal.
I deleted all other mods and still same problem. I don't even know since when it happens. yesterday it was ok. When I send driver from A - B just driving enerything is ok.
[log.txt](https://github.com/Courseplay/courseplay/files/2819242/log.txt)

**Comment by BI4LY** [comment]
Problem solved. There was problem with Holmer Terra Dos T4-40, it was almost full and stayed before the field (preparing to go new line but without worker, I didnt start him after back to game). When I started combine harvesting everything working good. But I have other problem. I wrote this as a comment in #3070,  but there is something else. When I start course with loading in Root Crop Storage mod I cant choose how much sugar beets load to transport. Every time he is loading full

**Comment by operinko** [comment]
Holmer Terra Dos T4-40 (and the ROPA TIGER 6 XL) with the Holmer HR 12 header are both bugged to hell and back as it is, regardless of other mods which could be affecting this.
The header tends to cause invisible walls when folding/unfolding.

**Comment by ThomasGaertner** [comment]
maybe its a case for @pvajko  ?

**Comment by pvajko** [comment]
fruit.lua has all over your name on it @ThomasGaertner :)

**Comment by ThomasGaertner** [comment]
guys, please dont mix the issues here . what  is the current issue ? 
- the nil error in fruit(193) ?
- loading roootcrop : is this GrainTransport mode ?
- header are invisible ? which mode ?
I will close here and if still current, please open a new issue for every topic
Thanks
